lib/spack/docs/module_file_support.rst

```diff
@@ -457,10 +457,10 @@ For instance, the following config options,
        tcl:
          all:
            suffixes:
-             ^python@3.12: 'python-3.12'
+             ^python@3: 'python{^python.version}'
              ^openblas: 'openblas'
 
-will add a ``python-3.12`` version string to any packages compiled with
+will add a ``python-3.12.1`` version string to any packages compiled with
```

Comment:
This change could be reverted with #38410 using `{^python.version.major}.{^python.version.minor}`.

Depending on which MR gets merged first, I will update the doc accordingly.